docs/pages/database-access/guides/rds.mdx

```diff
@@ -14,6 +14,11 @@ This guide will help you to:
   writing so it can't be used with Database Access.
 </Admonition>
 
+<Admonition type="note" title="MariaDB supported versions">
```

Comment:
The two back-to-back Admonitions are kind of hard to follow. I think we should include a single Admonition that starts with a general statement like, "The following products are not compatible with Database Access," followed by a two-item list that includes the products mentioned in these Admonitions.